pkg/kubelet/rkt/version.go

```diff
@@ -54,22 +54,13 @@ func (r rktVersion) Compare(other string) (int, error) {
 	return 0, nil
 }
 
-// checkVersion tests whether the rkt/systemd/rkt-api-service that meet the version requirement.
-// If all version requirements are met, it returns nil.
-func (r *Runtime) checkVersion(minimumRktBinVersion, recommendedRktBinVersion, minimumAppcVersion, minimumRktApiVersion, minimumSystemdVersion string) error {
-	// Check systemd version.
+func (r *Runtime) getVersions() error {
```

Comment:
Now `Version()` is called in goroutine https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1022.
While `Status()` is called in goroutine https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1025.
There is a data race here.